docs/source/topics/con_differences-from-production-openshift-install.adoc

```diff
@@ -4,10 +4,10 @@
 {rh-prod} is a regular OpenShift installation with the following notable differences:
 
 * **The {prod} OpenShift cluster is ephemeral and is not intended for production use.**
-* **There is no supported upgrade path to newer OpenShift versions.**
+* **{prod} does not have a supported upgrade path to newer OpenShift versions.**
```

Comment:
this might change now there is a SNO. We have to follow-up on this

/cc: @praveenkumar